### DISCUSSION TITLE
Fine-tuning  Model

### DISCUSSION THREAD

**Discussion by VitorCMatias** [mindee/doctr#1660] [Q&A]
Hello!
I’m using DocTR in a project, and I’d like to fine-tune the analysis to improve my results. I’m following the tutorials in the documentation. I can even generate the directory with the files: ['checkpoint', 'weights.data-...', 'weights.index'] using the scripts at  'references/recognition'. But how do I use these files in my system? Do I need to make other steps?

**Comment by felixdittrich92** [comment]
Hi @VitorCMatias :wave:,

Take a look at: https://mindee.github.io/doctr/using_doctr/custom_models_training.html#loading-your-custom-trained-model how to load custom trained models.

Best regards,
Felix
